packages/jest-cli/src/jest.js

```diff
@@ -50,17 +51,11 @@ const KEYS = {
   U: '75',
 };
 
-const printHeader = pipe => {
-  if (process.stdout.isTTY) {
-    pipe.write(chalk.bold.dim('Determining test suites to run...'));
-  }
-};
-
 const getMaxWorkers = argv => {
   if (argv.runInBand) {
     return 1;
   } else if (argv.maxWorkers) {
-    return argv.maxWorkers;
+    return parseInt(argv.maxWorkers, 10);
```

Comment:
at least it didn't pass the flow types. i think it was number when specified in package.json and string if from CLI, or something like that